dips/dip-10.md

```diff
@@ -101,130 +92,62 @@ struct DiemIdDomainEvent {
-
-* `Payment_Reference_ID` must be unique for each payment, it must not be reused in any way to avoid leaking privacy information on-chain.
-
-* `Payment_Reference_ID` returned by this endpoint is not signed for KYC purposes. This means that payment above KYC threshold can not be submitted right away using the provided Payment_Reference_ID. Instead, sender VASP needs to proceed to KYC endpoint to negotiate the signed Reference ID.
-
-* KYC endpoint API will be updated to accept `Payment_Reference_ID` returned by the Diem ID endpoint instead of address/subaddress.
 
-## User Object
-User Object defines reach information about the user.
-
-All fields, except for `diem_id`, in this object are optional - VASP can choose to share either of those fields or none.
+# End-to-End Experience
+Below is an example of using Diem ID domain for transferring money from one user to another. 
 
-Additionally, all fields in this object are informational - **they must not be used for KYC and only intended for UI purposes**.
-```json
-{
-   "diem_id": "<Diem_ID>",
-   "display_name": "<Display name>",
-   "profile_picture": "<profile_picture_uri>"
-}
+## Prerequisite:
+* VASPs get approval from association (via some offline process) on domain name
+* VASPs receive a CSV file from association that looks like this:
 ```
-
-| Field 	      | Type  | Required? 	| Description 	           |
-|-------	      |------ |-----------	|-------------	           |
-|diem_id         |str    | Y           | Diem ID of the user |
-|display_name     |str    | N           | Display name of the user. Usually a full name |
-|profile_picture  |str    | N           | Profile picture URL(see below) |
-
-If a user does not exist, VASP can choose to return an HTTP error, or the empty User Object, depending on it’s privacy choices.
-
-## Profile Picture URI
-Profile picture URI is a URI that represents a profile picture of a user. Wallet app must support two possible URI schemas for profile picture.
-
-* **Https schema.** If URI starts with https:, wallet must attempt to fetch profile picture from the HTTP URI provided. When fetching this URI wallet app will not provide any authentication. Wallet app will use GET method to fetch the profile picture specified using https schema.
-  * Ex.: *https://novi.com/pictures/alice.png*
-* **Data schema.** If URI starts with data:, wallet must interpret the URI as a [Data URL](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs) that contains profile picture encoded.
-  * Ex.: *data:image/gif;base64,AAAAAAAA*
-* **Unencrypted http** schema must not be used to share profile pictures.
+coinbase, 0xf72589b71ff4f8d139674a3f7369c69b
+novi, 0xc5ab123458df0003415689adbb47326d
+```
+* VASPs store the mapping internally
+
+## User Story: 
```

Comment:
prefer this to be really, really early like it was

also realized in reading and writing this, that we should be consistent in all our docs:
Bob's VASP is VASP B
Alice's VASP is VASP A

And we should probably have a pre-conditions to the user story:
* Alice's VASP, VASP A, has registered a DiemID domain at avasp
* Bob's VASP, VASP B, has registered a DiemID domain at bvasp

Reply:
but that gets deep into the weeds... hmm... something to chew on... technical versus overview

Reply:
Yeah, definitely believe we need to introduce this earlier so we can gloss over the technical details to make the rest of the pieces fit together

* User exchanges identifiers
* VASP does on-chain look up
* VASP performs off-chain protocol
* VASP submits on-chain transaction

Reply:
which then maps to the deep technical pieces

Reply:
ok agree a user flow at the beginning might be a good overview